src/components/LessonsList/index.tsx

```diff
@@ -0,0 +1,23 @@
+import {Grid, Typography} from '@mui/material';
+
+import {LessonCard} from '../LessonCard';
+
+import styles from './styles.module.scss';
+import {Lesson} from '../../../types';
+
+interface IProps {
+    lessons: Lesson[],
+    onClick: (id: string) => void;
+}
+
+export const LessonsList = ({lessons, onClick}: IProps) => {
+    return <Grid container direction="column" className={styles.lessons}>
+        <Typography mb={1} textAlign="center"><strong>Course lessons: </strong> </Typography>
+        {lessons
```

Comment:
If **lessons** undefined and if **lessons** array empty ?

Reply:
Component PageLoadStages is responsible for checking such cases.